example/l2fwd_simple/odp_l2fwd_simple.c

```diff
@@ -204,6 +206,23 @@ int main(int argc, char **argv)
 	global.if1 = create_pktio(argv[optind + 1], pool, &global.if1in,
 								&global.if1out);
 
+	/* Do some operations to increase code coverage in tests */
+	if (odp_pktio_mac_addr(global.if0, &correct_src, sizeof(correct_src))
+	    != sizeof(correct_src))
+		printf("Warning: can't get MAC address\n");
+	else if (memcmp(&correct_src, &global.src, sizeof(correct_src)) != 0)
+		printf("Warning: src MAC invalid\n");
+
+	odp_pktio_promisc_mode_set(global.if0, true);
+	odp_pktio_promisc_mode_set(global.if1, true);
+	(void)odp_pktio_promisc_mode(global.if0);
+	(void)odp_pktio_promisc_mode(global.if1);
```

Comment:
This should probably go to a separate PR. And ideally examples should not add much to code coverage on top of tests, so you might want also to add code to api testcases.

Reply:
The problem is that the pktio validation test case doesn't like an interface where packets are not being received. It fails a number of tests, and then for the pktio_test_recv_tmo it just hangs. With the validation tests, an entirely different test for a quiescent interface would be needed. Also, considering that this is testing a particular driver ("null"), not a particular API, I'm not entirely sure that API validation test would be the right location...